ssv/aggregator.go

```diff
@@ -76,7 +76,7 @@ func (r *AggregatorRunner) ProcessPreConsensus(signedMsg *SignedPartialSignature
 	// TODO waitToSlotTwoThirds
 
 	// get block data
-	res, err := r.GetBeaconNode().SubmitAggregateSelectionProof(duty.Slot, duty.CommitteeIndex, fullSig)
+	res, err := r.GetBeaconNode().SubmitAggregateSelectionProof(duty, fullSig)
```

Comment:
why make this change?

Reply:
cause it's needed more than those 2 vars from duty (CommitteeLength)

Reply:
Prysm sends slot, committee index, pub key and selection sig

Reply:
lets pass the exact params not the entire duty

Reply:
done